.prow.yaml

```diff
@@ -135,4 +183,19 @@ postsubmits:
               valueFrom:
                 secretKeyRef:
                   name: codecov-tokens
-                  key: ti-community-infra-tichi-codecov-token
\ No newline at end of file
+                  key: ti-community-infra-tichi-codecov-token
+  - name: pull-tichi-web-node14-lint
+    decorate: true
+    branches:
+      - ^master$
+    spec:
+      containers:
+        - image: node:14
+          command:
+            - /bin/bash
+            - -c
+            - >
+              cd ./web &&
+              npm ci &&
+              npm run build &&
```

Comment:
If it's just a lint check, why do we need to build?

Reply:
Since we don't have tests, we want to make sure we can compile them.
When I add the tests the task will be renamed to unit-test.

Reply:
OK